proto/src/determined/api/v1/api.proto

```diff
@@ -2219,4 +2219,15 @@ service Determined {
       tags: "Internal"
     };
   }
+
+  // Get a list of all unbound resource pools from the cluster
+  rpc GetUnboundResourcePools(GetUnboundResourcePoolsRequest)
+      returns (GetUnboundResourcePoolsResponse) {
+    option (google.api.http) = {
+      get: "/api/v1/resource-pools/unbound"
```

Comment:
Typically I would expect something like this to be a filter on the "get all of something" endpoint—e.g. `/api/v1/resource-pools?unbound=true` or something. Is there a particular reason for this to be its own, separate endpoint?

Reply:
hmm no particular reason. I can make `get resource pools` take in a boolean for whether or not it gets unbound? I'll test it and try